package.json

```diff
@@ -25,19 +26,26 @@
     "webpack-dev-server": "^4.0.0"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "1.6.1",
+    "@tippyjs/react": "4.2.5",
```

Comment:
We already have react-tooltip. Please consolidate into one tooltip library. I don't care which one.